hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
 
-	IAMSTATIC string = "${IAMSTATIC:-true}"
-	INITSHA1  string = "$DOCKER_INITSHA1"
-	INITPATH  string = "$DOCKER_INITPATH"
+	IAMSTATIC    string = "${IAMSTATIC:-true}"
+	INITSHA1     string = "$DOCKER_INITSHA1"
+	INITPATH     string = "$DOCKER_INITPATH"
+	EXPERIMENTAL   bool = $EXPERIMENTAL
```

Comment:
You mean kind of like we do with `IAMSTATIC`? :wink:

Also, this code wouldn't pass `gofmt`. :smile: